w_common/test/unit/disposable_common.dart

```diff
@@ -61,12 +50,8 @@ void testCommonDisposable(Func<StubDisposable> disposableFactory) {
       callback(dynamic argument, dynamic secondArgument),
       dynamic argument,
       dynamic secondArgument) {
-    test('should throw if called with a null first argument', () {
-      expect(() => callback(null, secondArgument), throwsArgumentError);
-    });
-
     test('should throw if called with a null second argument', () {
-      expect(() => callback(argument, null), throwsArgumentError);
```

Comment:
I think we should just delete this test as passing in `null` is disallowed my the method signature.